packages/now-cli/src/commands/deploy/latest.js

```diff
@@ -216,6 +217,9 @@ export default async function main(
     return 1;
   }
 
+  const hasLocalConfig = Boolean(localConfig);
+  localConfig = localConfig || {};
```

Comment:
I think we can remove those lines, and instead do all the `localConfig` reassignment after we have `rootDirectory`.